integration-tests/src/test/java/oracle/weblogic/kubernetes/ItConfigDistributionStrategy.java

```diff
@@ -767,7 +768,7 @@ private void verifyPodsStateNotChanged() {
   private void verifyIntrospectorRuns() {
     //verify the introspector pod is created and runs
     logger.info("Verifying introspector pod is created, runs and deleted");
-    String introspectPodName = domainUid + "-" + "introspect-domain-job";
+    String introspectPodName = getIntrospectPodName(domainUid);
```

Comment:
Minor nit: Would getIntrospectJobName would be more accurate? The runtime pod name is generated by k8s using a decoration of the JobName...

Reply:
LGTM now